server/nodemon.json

```diff
@@ -1,6 +1,7 @@
 {
   "ignore": ["**/*.test.ts", "**/*.spec.ts", ".git", "node_modules"],
   "watch": ["./"],
-  "exec": "ts-node ./src/app.ts",
-  "ext": "ts"
+  "exec": "node -r ts-node/register --inspect=127.0.0.1:9229 ./src/app.ts",
+  "ext": "ts",
+  "delay": 5000
```

Comment:
I don't like this delay since it slows down iteration speed. I'd rather we took it out.

What's the downside you've found with letting the server restart frequently? I haven't encountered any issues with it.

Reply:
The CPU fan goes out of control, it keeps restarting, reporting port in use, crashes, I have to restart it manually anyway but I will remove the setting. Overall, `nodemon` is not doing a good job at all.